### PR TITLE
Output PPCW when SWATINIT is enabled

### PR DESCRIPTION
**Pull request by totto82**

### CODE REVIEW

**Comment by totto82** [comment]
jenkins build this please

**Comment by totto82** [comment]
We need to add the extra output to the ref. as well

**Comment by totto82** [comment]
jenkins build this update_data please

**Comment by alfbr** [comment]
The three test failures are all the new model 1 cases complaining about PPCW. Have you had a look?

**Comment by totto82** [comment]
jenkins build this opm-tests=55 please

**Comment by alfbr** [comment]
Perfect :-)

**Comment by totto82** [comment]
jenkins build this opm-tests=55 please

**Comment by totto82** [comment]
jenkins build this update_data please

File: ebos/ecloutputblackoilmodule.hh [hunk 1]
```diff
@@ -713,8 +720,8 @@ public:
         if (FluidSystem::phaseIsActive(gasPhaseIdx) && saturation_[gasPhaseIdx].size() > 0) {
             sol.insert("SGAS", Opm::UnitSystem::measure::identity, std::move(saturation_[gasPhaseIdx]), Opm::data::TargetType::RESTART_SOLUTION);
         }
-        if (pcSwMdcOw_.size() > 0) {
-            sol.insert ("PPCW", Opm::UnitSystem::measure::identity, std::move(pcSwMdcOw_), Opm::data::TargetType::RESTART_SOLUTION);
+        if (ppcw_.size() > 0) {
+            sol.insert ("PPCW", Opm::UnitSystem::measure::pressure, std::move(ppcw_), Opm::data::TargetType::RESTART_SOLUTION);
         }
```
Comment (totto82) [inline ebos/ecloutputblackoilmodule.hh]: Ops. We still need the pcswmdcow for cases with hysteresis. Fix on its way.

Comment (totto82) [reply ebos/ecloutputblackoilmodule.hh]: fixed

**Comment by totto82** [comment]
jenkins build this update_data please

**Comment by totto82** [comment]
jenkins build this opm-tests=55 please

**Comment by totto82** [comment]
The tests failures seems totally unrelated. Is current master fine?

**Comment by totto82** [comment]
jenkins build this please

**Comment by andlaus** [comment]
> Is current master fine?

with yesterday's NTG related data updates it should be. (some other PRs have seen green after that.) possibly something else broke it again, though. I'll have a look.

**Comment by andlaus** [comment]
it seems like there is an additional field in the new results: the comparison for `mpi.compareECLFiles_flow+MSW_MODEL_1` produces the following line:

```
PPCW: 12         PPCW:  0 
```

I have to confess that I'm not completely sure what it means, though.

**Comment by totto82** [comment]
jenkins build this opm-tests=55 please

**Comment by andlaus** [comment]
all green, merging. Thanks.

**Pull request merged**